fieldservice_account_payment/__manifest__.py

```diff
@@ -9,7 +9,11 @@
     "author": "Open Source Integrators, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/field-service",
     "depends": [
-        "fieldservice_account",
+        # customer_id is a required field in fsm.location in fieldservice_account_analytic
+        # fieldservice_account_analytic is installed before fieldservice_account_payment
+        # so to avoid `null value in column "customer_id" violates not-null constraint` error
+        # we need to depend on fieldservice_account_analytic
+        "fieldservice_account_analytic",
```

Comment:
So the issue is only with unit tests, I presume?

I would try to avoid adding a possibly unwanted dependency.

Perhaps there's a different workaround, like a lazy-dependency during tests, that sets a default value through context?

Reply:
we could try

Reply:
Thanks !